src/runtime_src/driver/xclng/include/mailbox_proto.h

```diff
@@ -44,10 +45,72 @@ enum mailbox_request {
 
 struct mailbox_req_bitstream_lock {
 	pid_t pid;
-	xuid_t uuid;
+	u8 uuid[UUID_SZ];
 };
 
 
+enum group_kind {
+	SENSOR = 0,
```

Comment:
sounds like we need different cache policy for different groups? Only sensor data can be cached?

Reply:
chat w/ Sonal, it seems like we don't need different policy, but need a way to change the expiration period, say, through sysfs. User can disable cache and get real time data if they want.